pkgs/applications/networking/mkchromecast/default.nix

```
@@ -13,6 +13,7 @@
 , opusTools
 , gst_all_1
 , enableSonos ? true
+, qt5
```

```suggestion
, qtwayland
```

use `libsForQt5.callPackage` in `pkgs/top-level/all-packages.nix`

---

`libsForQt5.callPackage` is already used just didn't know ✌🏻